### PR TITLE
doc/development: update graphstrategy PNGs and improve steps how to generate them

### PR DESCRIPTION
**Pull request by rohieb**
The previous images did not reflect the example above, fix this.

### CODE REVIEW

**Comment by rohieb** [comment]
> > Merging [#591](https://codecov.io/gh/labgrid-project/labgrid/pull/591?src=pr&el=desc) into [master](https://codecov.io/gh/labgrid-project/labgrid/commit/a630074489ebefe869adaa3e96b1f2d59c8acd9c&el=desc) will **increase** coverage by `0.8%`.

Oh cool, I didn't know that writing docs could increase coverage :D

**Comment by jluebbe** [comment]
> > > Merging [#591](https://codecov.io/gh/labgrid-project/labgrid/pull/591?src=pr&el=desc) into [master](https://codecov.io/gh/labgrid-project/labgrid/commit/a630074489ebefe869adaa3e96b1f2d59c8acd9c&el=desc) will **increase** coverage by `0.8%`.
> 
> Oh cool, I didn't know that writing docs could increase coverage :D

There is some non-determinism in the coverage analysis that I've not been able to find.

**Comment by rohieb** [comment]
Mh, I still found some issues further down the document...

**Comment by rohieb** [comment]
Looks good to me now.

File: doc/development.rst [hunk 1]
```diff
@@ -314,43 +314,57 @@ Example
...
+The class can also render a graph as PNG (using GraphViz):
+
 .. code-block:: python
 
-    # render graph to png
-    >>> graph_strategy.graph.render('filename')
-    'filename.png'
+    >>> # use a test environment for this example
+    >>> import labgrid
+    >>> env = labgrid.environment.Environment('myplace.yaml')
```
Comment (Emantor) [inline doc/development.rst]: Can you add the example 'myplace.yaml' to the documentation as well?

**Comment by rohieb** [comment]
Right, good idea. Updated the docs once more.

File: doc/development.rst (hunk 1 above)
Comment (rohieb) [reply doc/development.rst]: Right, good idea. Updated the docs once more.

**Pull request merged**